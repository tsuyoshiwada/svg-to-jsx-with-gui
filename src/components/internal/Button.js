import styled from 'styled-components';

const Button = styled.button`
  display: inline-flex;
  justify-content: center;
  align-items: center;
  position: relative;
  margin: 0;
  padding: 0;
  border: none;
  border-radius: 0;
  font-weight: bold;
  font-family: inherit;
  vertical-align: middle;
  line-height: 1.4;
  text-align: center;
  text-decoration: none;
  cursor: pointer;
  user-select: none;
  transition: all 120ms ease-out;

  &:focus {
    outline: none;
  }
`;

export default Button;
