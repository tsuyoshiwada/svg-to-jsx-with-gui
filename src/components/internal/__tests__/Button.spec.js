import React from 'react';
import { shallow } from 'enzyme';
import Button from '../Button';


describe('components#<Button />', () => {
  test('Should be render', () => {
    const wrapper = shallow(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
