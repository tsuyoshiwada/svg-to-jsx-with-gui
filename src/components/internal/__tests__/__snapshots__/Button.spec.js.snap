// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components#<Button /> Should be render 1`] = `
<button
  className="sc-bdVaJa bfwJUA"
/>
`;
